charts/internal/shoot-dns-service-seed/templates/deployment.yaml

```diff
@@ -48,14 +48,18 @@ spec:
           - --target.id={{ .Values.seedId }}
           - --target.disable-deploy-crds
           - --target-owner-id={{ .Values.shootId }}-shootdns
-          - --controllers=dnssources,dnsentry-source,annotation
+          - --controllers=dnssources,dnsentry-source,annotation{{- if .Values.dnsProviderReplication.enabled -}},dnsprovider-replication{{- end }}
+          {{- if .Values.dnsProviderReplication.enabled }}
+          - --dnsprovider-replication.target-realms={{ .Release.Namespace }},
+          {{- end }}
           - --namespace=kube-system
           - --target-namespace={{ .Release.Namespace }}
           - --target-creator-label-name=gardener.cloud/shoot-id
           - --target-creator-label-value={{ .Values.creatorLabelValue }}
           - --target-set-ignore-owners
           - --dns-target-class=gardendns
           - --dns-class={{ .Values.dnsClass }}
+          - --lease-name=shoot-dns-service
```

Comment:
Does this require an rbac change?

Reply:
Currently the role covers all configmaps in the kube-system, so no change neeeded.
The dns-controller-manager still uses only configmaps for leases.